packages/sit-onyx/src/components/OnyxEmpty/OnyxEmpty.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts" setup>
+import circleX from "@sit-onyx/icons/circle-x.svg?raw";
+import OnyxIcon from "../OnyxIcon/OnyxIcon.vue";
+
+defineSlots<{
+  /**
+   * Label / text to display.
+   */
+  default(): unknown;
+  /**
+   * Optional slot to override the default icon.
+   */
+  icon?(): unknown;
+}>();
+</script>
+
+<template>
+  <div class="onyx-empty">
+    <slot name="icon">
+      <OnyxIcon :icon="circleX" size="48px" />
+    </slot>
+
+    <div class="onyx-empty__label onyx-text onyx-truncation-multiline">
+      <slot></slot>
```

Comment:
question: was a default text desired from UX?
if not, should / do we want to add a default empty text to our translations that is also used inside the storybook example? could be quite handy to spare the projects from translating that all the time ^^

Reply:
we should ask jannick if we want to align it with figma, then the figma component should also have a default text :)
Inside components like the table would also be totally fine ^^